IntroToCSharp/Shared/Components/Activities/HorizontalDivider.razor

```diff
@@ -0,0 +1 @@
+<MudDivider Class="mt-6 mb-6" DividerType="DividerType.FullWidth"></MudDivider>
```

Comment:
I don't think we need an entire component for this. Why nut just put this divider in there like this?